runtime/jcl/common/shared.c

```diff
@@ -1261,7 +1286,9 @@ Java_com_ibm_oti_shared_SharedClassURLHelperImpl_findSharedClassImpl3(JNIEnv* en
 	if (!getStringPair(env, &nameChars, &nameLen, &partitionChars, &partitionLen, classNameObj, partitionObj)) {
 		goto _errorPostClassNamePartition;
 	}
-	if (!(cpeType = getCpeTypeForProtocol((char*)urlElements.protocolChars, urlElements.protocolLen, urlElements.pathChars, urlElements.pathLen))) {
+	char* correctPath = NULL;
```

Comment:
This is C code. It may not compile on some platforms. Please move this to the beginning of the function.

Reply:
I C code, variables can be declared at the beginning of any block, not just at the top of a function. It should stay here.

Reply:
> It should stay here.

Is here the beginning of a block ?

Reply:
Sorry, @hangshao0 is correct: I misread the change thinking the `if` statement that's being removed introduced a new block.